swift/code.go

```diff
@@ -0,0 +1,35 @@
+package swift
+
+import (
```

Comment:
This was copy/pasted from the other languages, which is somewhat gross. But gives us a place to express literal swift syntax should we need. Also, happy to refactor this is the duplication is offensive. 

It would probably involve slicing up the interfaces a bit differently.